ui/components/PromptComponent.js

```diff
@@ -64,14 +64,17 @@ const styles = (theme) => ({
   },
 });
 
-const PromptActionButton = styled(Button)(({ theme, promptVariant }) => ({
+const PromptActionButton = styled(Button)(({ theme }) => ({
   margin: theme.spacing(0.5),
   padding: theme.spacing(1),
   borderRadius: 5,
-  backgroundColor: theme.palette.secondary[promptVariant || 'primary'],
+  backgroundColor: theme.palette.type === 'dark' ? '#00B39F' : '#607d8b',
   color: '#fff',
   '&:hover': {
-    backgroundColor: theme.palette.secondary[promptVariant || 'primary'],
+    backgroundColor:
+      theme.palette.type === 'dark'
+        ? theme.palette.secondary.focused
+        : theme.palette.secondary.focused,
```

Comment:
when we use same theme color you don't need to add a condition
```suggestion
    backgroundColor: theme.palette.secondary.focused,
```

Reply:
hey @Yashsharma1911 i have changes now  plz review sorry for intial mistakes!!

Reply:
Ohh good, it would be helpful if you can share video or screenshot in both light and dark mode of changes

Reply:
sure !
![image](https://github.com/meshery/meshery/assets/128807866/0936e4d8-cf5d-4b75-90e5-fdde43d2e3f8)
![image](https://github.com/meshery/meshery/assets/128807866/9e4679e1-9704-4943-87b5-6e53f7c01e70)

Reply:
[Screencast from 19-11-23 12:59:43 PM IST.webm](https://github.com/meshery/meshery/assets/128807866/78e79adc-ce32-4ec0-ae0f-8d62a5521ed8)

Reply:
hey  @Yashsharma1911 plz have a look.